### ISSUE TITLE
Dockerfile obsolete

### ISSUE THREAD

**Issue by dark-red-velvet** [uvdesk/community-skeleton#266]
While building from docker, Step 5 and 11 fails

```
Package libapache2-mod-php7.2 is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'libapache2-mod-php7.2' has no installation candidate
E: Unable to locate package php7.2-imap
E: Couldn't find any package by glob 'php7.2-imap'
E: Couldn't find any package by regex 'php7.2-imap'
E: Unable to locate package php7.2-mysql
E: Couldn't find any package by glob 'php7.2-mysql'
E: Couldn't find any package by regex 'php7.2-mysql'
```

```
Step 11/14 : RUN     a2enmod php7.2 rewrite;     chmod +x /usr/local/bin/uvdesk-entrypoint.sh;     dpkgArch="$(dpkg --print-architecture | awk -F- '{ print $NF }')";     wget -O /usr/local/bin/gosu "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch"     && wget -O /usr/local/bin/gosu.asc "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch.asc";     export GNUPGHOME="$(mktemp -d)"     && gpg --batch --keyserver hkps://keys.openpgp.org --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 	&& gpg --batch --verify /usr/local/bin/gosu.asc /usr/local/bin/gosu     && gpgconf --kill all     && chmod +x /usr/local/bin/gosu     && gosu nobody true;         wget -O /usr/local/bin/composer.php "https://getcomposer.org/installer";     actualSig="$(wget -q -O - https://composer.github.io/installer.sig)";     currentSig="$(shasum -a 384 /usr/local/bin/composer.php | awk '{print $1}')";     if [ "$currentSig" != "$actualSig" ]; then         echo "Warning: Failed to verify composer signature.";         exit 1; 	fi;     php /usr/local/bin/composer.php --quiet --filename=/usr/local/bin/composer     && chmod +x /usr/local/bin/composer;     chown -R uvdesk:uvdesk /var/www;     rm -rf         "$GNUPGHOME"         /var/lib/apt/lists/*         /usr/local/bin/gosu.asc         /usr/local/bin/composer.php         /var/www/bin         /var/www/html         /var/www/uvdesk/.docker;
 ---> Running in 26b9cf91aafc
/bin/sh: 1: a2enmod: not found
/bin/sh: 1: wget: not found
gpg: keybox '/tmp/tmp.gkmsvmHupz/pubring.kbx' created
gpg: /tmp/tmp.gkmsvmHupz/trustdb.gpg: trustdb created
gpg: key 036A9C25BF357DD4: public key "Tianon Gravi <tianon@tianon.xyz>" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg: can't open '/usr/local/bin/gosu.asc': No such file or directory
gpg: verify signatures failed: No such file or directory
/bin/sh: 1: wget: not found
/bin/sh: 1: wget: not found
/bin/sh: 1: shasum: not found
/bin/sh: 1: php: not found
```

**Comment by dark-red-velvet** [comment]
Solved this by using ubuntu:18.04 image instead of ubuntu:latest

**Comment by papnoisanjeev** [comment]
@dark-red-velvet 
Glad to know that your issue has been resolved.
